src/styles/blocks/card.scss

```diff
@@ -0,0 +1,98 @@
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  display: flex;
+  flex-direction: column;
+  align-content: space-between;
+  border: 1px solid #e5e5e5;
+  background-color: #fff;
+
+  &:not(:nth-child(4n)) {
+    margin-right: $card-gap-side;
+  }
+
+  &:nth-child(n+5) {
+    margin-top: $card-gap-top;
+  }
+
+  &__image {
+    margin: 32px 19px 0;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__info {
+    margin: 40px 16px 16px;
+    display: flex;
+    flex-direction: column;
+  }
+
+  &__name {
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+    color: $color-main-accent;
+  }
+
+  &__product-code {
+    margin-top: 4px;
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $color-secondary;
+  }
+
+  &__reviews {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin-top: 16px;
+  }
+
+  &__reviews-text {
+    font-weight: 400;
+    font-size: 10px;
+  }
+
+  &__price {
+    margin-top: 24px;
+    display: flex;
+    justify-content: space-between;
+  }
+
+  &__price-label {
+    font-weight: 400;
+    font-size: 12px;
+    line-height: 18px;
+    color: $color-secondary;
+  }
+
+  &__price-number {
+    font-weight: 700;
+    line-height: 18px;
+    color: $color-main-accent;
+  }
+
+  &__buy-link {
+    box-sizing: border-box;
+    margin-top: 16px;
+    background-color: $color-blue-accent;
+    color: #fff;
```

Comment:
this repeatable color can be moved to variables too

Reply:
Thanks, now it is done)